### PR TITLE
fix: Virtual Events: Open "Join video" in same tab for "in-house" streams

### LINKED ISSUE #6172: Virtual Events: Open "Join video" in same tab for "in-house" streams
Please implement the following for "Join Video" links:
* Open "Join Video" in the same tab for streams "in-house" streams with our side panel. Show external streams in another tab.

Implement this in the following places:
1. On the right sidebar "Join Video" link
2. For the "Join Video" links in the schedule

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #6172

#### Short description of what this resolves:
1.Opens "Join Video" in the same tab for "in-house" streams. 
2.Shows external streams in another tab.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
As the logic is repeated, better make it a getter in VideoStream model - `isInternalStream` and use elsewhere

File: app/components/public/session-item.js [hunk 1]
```diff
@@ -56,8 +56,8 @@ export default class SessionItem extends Component {
   @action
   goToStream() {
     const url = this.router.urlFor('public.stream.view', this.args.event?.identifier ?? this.args.session.get('event.identifier'), this.args.session.get('microlocation.videoStream.slugName'), this.args.session.get('microlocation.videoStream.id'));
-    const provider = this.args.session.get('microlocation.videoStream.videoChannel.provider');
-    if (provider === 'jitsi' || provider === 'bbb' || this.args.sameTab) {
+    const provider = this.args.session.get('microlocation.videoStream.videoChannel.isInternalStream');
+    if (provider || this.args.sameTab) {
```
Comment (iamareebjamal) [inline app/components/public/session-item.js:60]:
```suggestion
    const isInternal = this.args.session.get('microlocation.videoStream.videoChannel.isInternalStream');
    if (isInternal || this.args.sameTab) {
```

**Comment by iamareebjamal** [comment]
isInternalStream should be in stream, not channel model

**Comment by MeghalBisht** [comment]
Internal streams are still opened in a new tab when I add the check in stream model but works fine with channel model.

**Pull request merged**